frontend/openchat-client/src/openchat.ts

```diff
@@ -3454,12 +3454,10 @@ export class OpenChat extends OpenChatAgentWorker {
     }
 
     async threadPreviews(
-        chatId: ChatIdentifier | undefined,
+        _chatId: ChatIdentifier | undefined,
```

Comment:
Surely this can be removed?

Reply:
It's a bit of a hack. It's passed in so that the fn will get called reactively if the chatId changes. 🤮